app/controllers/api/database.php

```diff
@@ -281,6 +285,300 @@
         $response->noContent();
     });
 
+App::post('/v1/database/collections/:collectionId/attributes')
+    ->desc('Create Attribute')
+    ->groups(['api', 'database'])
+    ->label('event', 'database.attributes.create')
+    ->label('scope', 'attributes.write')
+    ->label('sdk.namespace', 'database')
+    ->label('sdk.platform', [APP_PLATFORM_SERVER])
+    ->label('sdk.method', 'createAttribute')
+    ->label('sdk.description', '/docs/references/database/create-attribute.md')
+    ->label('sdk.response.code', Response::STATUS_CODE_CREATED)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
+    ->label('sdk.response.model', Response::MODEL_ATTRIBUTE)
+    ->param('collectionId', null, new UID(), 'Collection unique ID. You can create a new collection using the Database service [server integration](/docs/server/database#createCollection).')
+    ->param('id', null, new Text(256), 'Attribute ID.')
+    ->param('type', null, new Text(256), 'Attribute type.')
+    ->param('size', null, new Integer(), 'Attribute size.')
+    ->param('required', null, new Boolean(), 'Is required flag.')
```

Comment:
```suggestion
    ->param('required', null, new Boolean(), 'Is required.')
```

Don't like the word flag, can we think of a better description?